src/Client.php

```diff
@@ -359,9 +359,12 @@ public function send($method, $payload, $path, $params = [], array $headers = []
 
         if ($response->getStatusCode() === 422) {
             $content = json_decode($response->getBody()->getContents(), true);
-            $content = $content['details'] ?? [];
 
-            throw new Http\Exception\UnprocessableEntityException($content);
+            if (isset($content['invalidFields']) && is_array($content['invalidFields'])) {
```

Comment:
@2e3s Are you sure? I'm not sure what API may return.

Reply:
But in case of we always can use DataValidationException instead of UnprocessableEntityException, then we may remove UnprocessableEntityException there.

Reply:
Am I sure that API returns `invalidFields` as well as `details` in case of 422? Yes